### DISCUSSION TITLE
Misspelled "docker-compose,yml" at https://www.authelia.com/configuration/methods/files/#docker-compose

### DISCUSSION THREAD

**Discussion by hvsharma** [authelia/authelia#8392] [Documentation Feedback] [answered]
<img width="881" alt="image" src="https://github.com/user-attachments/assets/301e2bf2-bd7b-4177-ae65-814eff003b0c">

**Comment by hendrik1120** [comment, marked as answer]
Thanks for the report, it will be fixed shortly
